salt/metalk8s/map.jinja

```diff
@@ -36,6 +36,14 @@
         },
       },
     },
+    'default': {
+      'default': {
+        'kubernetes': versions.kubernetes,
+        'repo': {
+          'images': versions.images,
+        },
+      },
+    },
```

Comment:
Instead of doing this, cannot we set the grains directly in the container, so that it has the same behavior as what we do for salt-minion

Basically, just create the `/etc/salt/grains` file like that 
```yaml
os_family: RedHat
os: CentOS
```
(And add a comment to make sure we remove it once we move to salt 3004)

Reply:
I'll give it a try